.github/workflows/publish_linux_release_candidate.yml

```diff
@@ -0,0 +1,70 @@
+name: Publish Linux Release Candidate
```

Comment:
I've chosen `publish` as the reserved word here instead of `release` as I don't want to confuse the build mode (`release`) with the action of publishing. 

Without this distinction, would "releasing" the binary mean building a release binary, or to actually push that binary to production?

This also required me to rename some scripts:

I've also renamed things so they're more consistent, we now have:
```
build, package_common => build_common
package_linux, package_windows => build_linux, build_windows
release_linux, release_windows => build_linux_release, build_windows_release
```